crates/neon-runtime/src/napi/bindings/functions.rs

```diff
@@ -336,6 +336,38 @@ mod napi6 {
             ) -> Status;
 
             fn get_instance_data(env: Env, data: *mut *mut c_void) -> Status;
+
+            fn create_bigint_int64(env: Env, value: i64, result: *mut Value) -> Status;
+
+            fn create_bigint_uint64(env: Env, value: u64, result: *mut Value) -> Status;
+
+            fn create_bigint_words(
+                env: Env,
+                sign_bit: isize,
+                word_count: usize,
+                words: *mut u64,
```

Comment:
```suggestion
                words: *const u64,
```

Also, `result` is missing. I recommend using the `bindings.rs` output from `nodejs-sys` as a reference.

```sh
cat ./target/debug/build/nodejs-sys-*/out/bindings.rs
```